iac-extensions/terraform/src/test/resources/checks/Azure/SubscriptionOwnerCapabilitiesCheck/compliant.tf

```diff
@@ -33,6 +33,6 @@ resource "azurerm_role_definition" "role-with-limited-assignable-scopes-alternat
   }
 
   assignable_scopes = [
-    "/subscriptions/${var.subscription_id}/resourceGroups/${var.environment_name}" # Limited to a resource group
+    "/subscriptions/${data.azurerm_subscription.primary.id}/resourceGroups/${var.environment_name}" # Limited to a resource group
```

Comment:
I don't fully understand why did you change this existing test

Reply:
Sorry for the lack of explanation. The content of this check is to check if a relation is created to a resource and contains the keyword `resourceGroups` in it's path. The focus here is on the keyword `resourceGroups`. I had to change the test, because we don't resolve variables and therefore I entered the resolved relation of the resource to be checked. If the keyword would be missing, then we would have a false negative, or the keyword check would not work, because the relation is not provided as a basic requirement.

Reply:
Ok I see, thanks for clarifying!